apps/learner/src/lib/components/LearningUnit.svelte

```diff
@@ -59,18 +59,16 @@
   </div>
 
   {#if showplayerpanel}
-    <div class="flex items-center justify-between">
+    <div class="flex items-center gap-x-3">
       <button
-        class="flex cursor-pointer items-center gap-x-2 rounded-full bg-slate-100 px-6 py-4 transition-colors hover:bg-slate-200"
+        class="border-1 flex cursor-pointer items-center gap-x-2 rounded-full border-black/10 bg-slate-100 bg-white/50 px-6 py-4 backdrop-blur-[20px] transition-colors hover:bg-slate-200"
```

Comment:
I think we can just use `border` instead of `border-1`. For the border color, can use `border-slate-950/10` instead.

Why do we need `backdrop-blur`?